src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateManaged/InflaterManaged.cs

```diff
@@ -140,16 +142,27 @@ public int Inflate(byte[] bytes, int offset, int length)
                     if (_hasFormatReader)
                     {
                         Debug.Assert(_formatReader != null);
-                        _formatReader.UpdateWithBytesRead(bytes, offset, copied);
+                        // Skip the copy if bytes is actually an array.
```

Comment:
This is all dead code.  _formatReader is always going to be null.

Reply:
https://github.com/dotnet/runtime/pull/47408

Reply:
That's awesome! I think what I'm going to do is wait until #47408 is merged, and then I'll redo this pull-request. That way we'll avoid merge conflicts.